### ISSUE TITLE
[App Icons Improvements] VoiceOver Does Not Announce the Selected App Icon in App Icon Settings - iOS 15

### ISSUE THREAD

**Issue by data-sync-user** [mozilla-mobile/firefox-ios#25460]
## Steps to reproduce
1. Enable VoiceOver on the device (Settings → Accessibility → VoiceOver).
1. Open the app and navigate to Settings → App Icon.
1. Swipe through the available app icon options using VoiceOver gestures.
1. Listen to the VoiceOver announcement.

### **Expected** behavior:
VoiceOver should announce the selected app icon 

**Actual behavior:**

The selected app icon is not announced. Not able to reproduce this issue using iOS 16.x, 17.x nor 18.x 

### Device & build information
* Device: iPhone 6s (iOS 15.7)
* Build version: v137 (52367)

┆Issue is synchronized with this [Jira Bug](https://mozilla-hub.atlassian.net/browse/FXIOS-11686)

**Comment by data-sync-user** [comment]
➤ Isabella commented:

Since we’re looking at bumping our minimum deployment target soon, this is a low priority issue cc Andres Furlan
